spec/sanitized_file_spec.rb

```diff
@@ -9,6 +9,7 @@
 
 describe CarrierWave::SanitizedFile do
   before do
+    FileUtils.rm_rf(file_path("new_dir"))
```

Comment:
Hooking into `after` might be better, where cleanup operations are supposed to be.